packages/bootstrap-4/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -12,15 +12,30 @@ const FieldTemplate = ({
   rawErrors = [],
   rawHelp,
   rawDescription,
+  label,
+  schema,
+  required
 }: FieldTemplateProps) => {
+
+  const shouldDisplayLabel = displayLabel && (label || schema.title);
```

Comment:
You can use the `registry.schemaUtils.getDisplayLabel(schema, uiSchema)` utility here.